curriculum/challenges/english/10-coding-interview-prep/rosetta-code/hash-from-two-arrays.md

```diff
@@ -8,7 +8,7 @@ dashedName: hash-from-two-arrays
 
 # --description--
 
-Using two arrays of length, create a hash object where the elements from one array (the keys) are linked to the elements of the other (the values).
+Using two arrays' length, create a hash object where the elements from one array (the keys) are linked to the elements of the other (the values).
```

Comment:
```suggestion
Given two arrays, create a hash object where the elements from one array (the keys) are linked to the elements of the other (the values).
```
This doesn't quite make sense to me. It seems like this is saying to use the length of the arrays - which you do, but not in the context of this sentence.